semcore/base-trigger/src/FilterTrigger.jsx

```diff
@@ -188,32 +196,48 @@ class ClearButton extends Component {
 
   render() {
     const SFilterTrigger = Root;
-    const { styles, empty, size, disabled, getI18nText, uid, id } = this.asProps;
+    const { styles, empty, size, disabled, getI18nText } = this.asProps;
 
     if (empty) return null;
 
     return sstyled(styles)(
-      <SFilterTrigger
-        render={BaseTrigger}
-        tag='button'
-        type='button'
-        size={size}
-        empty={empty}
-        selected
-        disabled={disabled}
-        id={`igc-${uid}-filter-trigger-clear`}
-        aria-labelledby={`igc-${uid}-filter-trigger-clear ${id}`}
-        aria-label={getI18nText('clear')}
-      >
-        <FilterTrigger.Addon tag={Close} />
-      </SFilterTrigger>,
+      <Hint>
+        <Hint.Trigger>
+          <SFilterTrigger
+            render={BaseTrigger}
+            tag='button'
+            type='button'
+            size={size}
+            empty={empty}
+            selected
+            disabled={disabled}
+            title={getI18nText('clear')}
```

Comment:
```suggestion
            title={getI18nText('clear')}
```
can we replace title with aria-label?